docs/user/dql/functions.rst

```diff
@@ -1458,6 +1458,14 @@ Example::
     | 239                             |
     +---------------------------------+
 
+    os> SELECT DAY_OF_YEAR(DATE('2020-08-26'))
+    fetched rows / total rows = 1/1
+    +-----------------------------------+
+    | DAY_OF_YEAR(DATE('2020-08-26'))   |
+    |-----------------------------------|
+    | 239                               |
+    +-----------------------------------+
+
```

Comment:
Can we add samples for datetime objects and show that it can get dayofyear with those too please.

Reply:
Added in https://github.com/Bit-Quill/opensearch-project-sql/pull/173/commits/226c1a58d51cc544925bb59cc65f9bbf38863105 and https://github.com/Bit-Quill/opensearch-project-sql/pull/173/commits/f1ed83261c96515f29fae59c500fee844cf23820